codespan-lsp/src/lib.rs

```diff
@@ -77,7 +77,7 @@ where
     })
 }
 
-pub fn character_to_line_offset(line: &str, character: u64) -> Result<usize, Error> {
+pub fn character_to_line_offset(line: &str, character: u32) -> Result<usize, Error> {
```

Comment:
This would need a `0.11.0` version bump, because it's changing an external type signature.

Reply:
I'm also unsure if this works for the prior versions of `lsp-types`? Like, it might need to be updated to just `>=0.84.0, <0.85`.

Reply:
I am not sure this should even be part of the public interface.

Couldn't `>=0.84.0, <0.85` be written simply as `0.84`?

Reply:
Also this is more or less necessary because of the usage in line 121 with the changed data type.

Reply:
`>=0.84.0, <0.85` would mean that we can expand the version range more easily in the future, in a backwards-compatible way.

Reply:
And yeah, it's ok if we change the types, just means we need to bump to `v0.11.0` in the version-bump PR.